plugin/src/main/java/org/opensearch/ml/rest/RestMLDeleteConnectorAction.java

```diff
@@ -45,8 +51,8 @@ public List<Route> routes() {
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
         String connectorId = request.param(PARAMETER_CONNECTOR_ID);
-
-        MLConnectorDeleteRequest mlConnectorDeleteRequest = new MLConnectorDeleteRequest(connectorId);
+        String tenantId = getTenantID(mlFeatureEnabledSetting.isMultiTenancyEnabled(), request);
```

Comment:
Can this setting be modified by users?

Reply:
This won't be a dynamic settings [[ref](https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java#L272-L303)]

Reply:
Got it, thanks for the ref.